server/service.go

```diff
@@ -131,11 +131,12 @@ func (m *BoostService) StartHTTPServer() error {
 		Addr:    m.listenAddr,
 		Handler: m.getRouter(),
 
-		ReadTimeout:       0,
-		ReadHeaderTimeout: 0,
+		ReadTimeout:       600 * time.Millisecond,
```

Comment:
may be too low, in case it's over the network? 🤔

Reply:
maybe you can make it configurable as an env var?

Reply:
done, please recheck